app/views/maps/view_map.html.erb

```diff
@@ -0,0 +1,104 @@
+        <i class="fas fa-pencil-alt"> </i> Edit this map
+    </button>
+</a>
+<div class="container-fluid">
+  <div class="text-center lead-text">
+  <p class="text-muted"> a Community Atlas of </p>
+  <h2><%= @map.location %> </h2>
+  <p> <%= link_to "@#{@map.author}", "/profile/#{@map.author}" unless @map.author == 'anonymous' %> </p>
+  </div>
+  <div class="row text-center">
+      <div class="col-md-4 offset-md-4 text-block">
+          <img class="mx-auto d-block" style="max-height: 75px; max-width: 75px;" src="<%= asset_path('balloon1.png') %>">
+          <h3>Make your own map </h3>
+          <p>Use a kite, balloon, pole or drone to take an aerial photo and tell your own visual story of the place </p>
+          <br> <br>
+          <p>
+          <a class="btn btn-primary btn-lg" href= 'https://store.publiclab.org/collections/mapping-kits'>
+              Get a mapping kit<i class="fa fa-angle-double-right fa-fw" style="font-size:18px;color:white;"></i>
+          </a>
+          <a class="btn btn-primary btn-lg" href="<%= new_map_url %>">
+              Add a new map <i class="fa fa-plus fa-fw" style="font-size:12px;color:white;"></i>
+          </a>
+          </p>
+
+      </div>
+  </div>
+</div>
+  <br><br>
+<div class="all-maps light-blue">
+    <br>
+    <h3 class="text-center"> Nearby Maps </h3>
+         <hr style="max-width: 500px;">
+    <br>
+    <%= render :partial => 'front_ui/maps', :locals => { :maps => @maps } %>
+</div>
+  <br><br>
+
+
+  <div class="container-fluid">
+      <div class="row text-center lead-text" id="info">
+          <div class="col-md-3 col-sm-5 offset-2">
+              <img class="mx-auto d-block" style="max-height: 75px; max-width: 75px;" src="<%= asset_path('quiz.png') %>">
+              <br>
+              <h3>Ask a question</h3>
+          </div>
+          <div class="col-md-3 col-sm-5 offset-2">
+              <img class="mx-auto d-block faded" style="max-height: 75px; max-width: 75px;" src="<%= asset_path('binoculars.png') %>">
+              <br>
+              <h3>Projects/ Requests for observations</h3>
+          </div>
+      </div>
+      <div class="row text-center lead-text" id="info">
+          <div class="col-md-3 col-sm-5 offset-2">
+              <p> Get help from others and share ideas with the PublicLab Community</p>
+              <br>
+          </div>
+          <div class="col-md-3 col-sm-5 offset-2">
+              <p> Community observations and monitoring are a key source of environmental data. Help add to one of the [projects] here. Read more or post your oen call for input</p>
+              <br>
+          </div>
+      </div>
+      <div class="row text-center lead-text" id="info">
+          <div class="col-md-3 col-sm-5 offset-2">
+              <p> <%= link_to "Ask a question", 'https://publiclab.org/questions/new', target: 'blank', class: "btn btn-lg btn-primary" %></p>
```

Comment:
Can we do a URL like: 

```https://publiclab.org/questions/new?tags=lat:<%= @map.lat %>,lon:<%= @map.lon %>&body=Question posted from map at https://mapknitter.org/m/<%= @map.slug %>```

Reply:
and for the "post an observation" as well, below?

Reply:
Yeah, this looks great..will try it. Thanks Jeff